packages/content/lib/index.js

```diff
@@ -138,12 +156,20 @@ module.exports = async function (moduleOptions) {
 
   /* istanbul ignore if */
   if (isSSG) {
+    // Create a hash to fetch the database
+    const dbHash = hash(JSON.stringify(database.items._data)).substr(0, 8)
+    // Pass the hash to the publicRuntimeConfig to be used in client side
+    this.options.publicRuntimeConfig._CONTENT_HASH_DB = dbHash
```

Comment:
runtimeConfig is a js object not env to be uppercase. We may use `.publicRuntimeConfig.content = { dbHash }`

Reply:
Agree :)

Reply:
ok, so I change this !